pages/posts/roadmap-2024.md

```diff
@@ -0,0 +1,219 @@
+    -  Key result 3:
+        - \>=1 application or client using Portal State network to request `eth_getProof` data
+
+- **Objective 4: Develop and maintain reliable tooling**
+    - Key result 1: 
+        - Glados maintains 99% uptime and is able to handle high volume of traffic (up to 10^5 visits/day)
+    - Key result 2:
+        - Portal Hive has 100% test coverage for all core functionality for all three networks across all three flagship Portal clients
+
+
+- **Objective 5: Communicate effectively with users and developers**
+    - Key result 1: 
+        - Github issues and comments across the Portal specs, Glados, and client repositories are always responded to (even if just an acknowledgement of receipt) in less than 3 days.
+    - Key result 2: 
+        - New features and milestones are reported via the Portal blog within 1 month of final PR meging merged.
+    - Key result 3:
+        - At least 5 presentations focused on portal Network are delivered at major Ethereum events in 2024, including Devcon.
+
+
+### 2025 and beyond
+
+These are longer term objectives that might be researched and discussed during 2024 but are not expected to be shipped until at least 2025. These will be revisited during 2025 roadmap planning.
+
+- **Objective 1: Portal Network is widely accepted as the default source for light client execution and consensus data**
+    - \> 10 apps/light clients have replaced third party centralized RPC providers with Portal Network.
+
+- **Objective 2: Portal Network is widely considered critical Ethereum infrastructure** 
+    - Key result 1: Portal mode is available is most Ethereum execution clients, either enablign them to aggressively prune their local data or to accelerate their sync times.
+
+- **Objective 3: Portal Network is supporting major new developments for Ethereum**
+
+    - Key result 1: Portal Network serves data enabling e.g. stateless clients, SNARKified EVM, or other substantial Ethereum upgrades from the protocol roadmap. 
+
+- **Objective 4: Portal developers ship major new features that change how normal users interact with Ethereum**
+
+    - Key result 1: Portal team have beta implementations of canonical transaction index network
+    
+    - Key result 2: Portal team have agreed and begun to implement a design for a two-way bridge to Ethereum to enable transaction forwarding from Poral --> Ethereum
+
+
+## Where are we now?
+
+The Portal Network has existed for 5 years, during which time there have been substantial course corrections and redesigns in response tothe evolving wider Ethereum landscape. Today, Portal Network is not widely used, but it is on the precipice of launching highly impactful infrastructure that will benefit large parts of the Ethereum ecosystem. 2024 is expected to be a pivotal year, where we focus on moving what were previously R&D projects into production and focusing on growing adoption for our core features: the History network, State network and Beacon network.
```

Comment:
Is it not rather about 3 years? And before focus was rather statelessness and beam sync?

Reply:
I am not sure - I guess the start is fuzzy as it grew from other earlier projects. @pipermerriam ?